### PR TITLE
Added fullscreen button in Physics activity

### PR DESCRIPTION
**Pull request by jontymagan**
Added icons
Added css for buttons
Updated index.html
Updated activity.js

### CODE REVIEW

File: activities/PhysicsJS.activity/css/activity.css [hunk 1]
```diff
@@ -1,63 +1,91 @@
 #main-toolbar #activity-button {
   background-image: url(../activity/activity-icon.svg);
-  z-index: 3;
```
Comment (llaske) [inline activities/PhysicsJS.activity/css/activity.css]: Not sure to understand why you need to change z-index

File: activities/PhysicsJS.activity/js/activity.js [hunk 2]
```diff
@@ -540,3 +542,14 @@ define(["sugar-web/activity/activity"], function (activity) {
     });
 
 });
+// Full screen
```
Comment (llaske) [inline activities/PhysicsJS.activity/js/activity.js]: You should define this into the body of the domReady! handler.

File: activities/PhysicsJS.activity/css/activity.css [hunk 3]
```diff
@@ -98,4 +116,8 @@ canvas {
 	#main-toolbar #stop-button {
 		position: absolute;
 	}
+
+	#unfullscreen-button{
+    right: 15%;
```
Comment (ashish0910) [inline activities/PhysicsJS.activity/css/activity.css]: Please fix indentation issue here

File: activities/PhysicsJS.activity/js/activity.js [hunk 4]
```diff
@@ -128,6 +128,8 @@ define(["sugar-web/activity/activity"], function (activity) {
 				currentType = -1;
 				switchToType(currentType);
 			}, true);
+
```
Comment (ashish0910) [inline activities/PhysicsJS.activity/js/activity.js]: Remove extra blank lines here

**Pull request closed without merging**